docs/source/configuration/advanced_configuration.md

```diff
@@ -4,8 +4,52 @@ The documentation on [configuration](./configuration_basics.md) describes how to
 
+## OmegaConfigLoader
+
+[OmegaConf](https://omegaconf.readthedocs.io/) is a Python library tailored for managing settings. It operates as a YAML-based hierarchical system for organizing configurations and supports merging configurations from various sources.
+
+From Kedro 0.18.5 you can use the [`OmegaConfigLoader`](/kedro.config.OmegaConfigLoader) which uses `OmegaConf` to load data.
+
+```{note}
+`OmegaConfigLoader` is under active development. It was first available from Kedro 0.18.5 with additional features due in later releases. Let us know if you have any feedback about the `OmegaConfigLoader`.
+```
+
+`OmegaConfigLoader` can load `YAML` and `JSON` files. Acceptable file extensions are `.yml`, `.yaml`, and `.json`. By default, any configuration files used by the config loaders in Kedro are `.yml` files.
+
+To use `OmegaConfigLoader` in your project, set the `CONFIG_LOADER_CLASS` constant in your [`src/<package_name>/settings.py`](../kedro_project_setup/settings.md):
+
+```python
+from kedro.config import OmegaConfigLoader  # new import
+
+CONFIG_LOADER_CLASS = OmegaConfigLoader
+```
+### Advanced `OmegaConfigLoader` features
+Some advanced use cases of `OmegaConfigLoader` are listed below:
+- [Advanced configuration](#advanced-configuration)
+  - [OmegaConfigLoader](#omegaconfigloader)
+    - [Advanced `OmegaConfigLoader` features](#advanced-omegaconfigloader-features)
+  - [TemplatedConfigLoader](#templatedconfigloader)
+    - [Provide template values through globals](#provide-template-values-through-globals)
+  - [Advanced Kedro configuration](#advanced-kedro-configuration)
+    - [How to change which configuration files are loaded](#how-to-change-which-configuration-files-are-loaded)
+    - [How to ensure non default configuration files get loaded](#how-to-ensure-non-default-configuration-files-get-loaded)
+    - [How to bypass the configuration loading rules](#how-to-bypass-the-configuration-loading-rules)
+    - [How to use Jinja2 syntax in configuration](#how-to-use-jinja2-syntax-in-configuration)
+    - [How to do templating with the `OmegaConfigLoader`](#how-to-do-templating-with-the-omegaconfigloader)
+      - [Parameters](#parameters)
+      - [Catalog](#catalog)
+      - [Other configuration files](#other-configuration-files)
+    - [How to use global variables with the `OmegaConfigLoader`](#how-to-use-global-variables-with-the-omegaconfigloader)
+    - [How to use resolvers in the `OmegaConfigLoader`](#how-to-use-resolvers-in-the-omegaconfigloader)
+    - [How to load credentials through environment variables](#how-to-load-credentials-through-environment-variables)
+
+
 ## TemplatedConfigLoader
 
+```{note}
+`ConfigLoader` and `TemplatedConfigLoader` have been deprecated since Kedro `0.18.12` and will be removed in Kedro `0.19.0`. Refer to the [Migration guide for config loaders](./config_loader_migration.md) for instructions on how to update your code base to use `OmegaConfigLoader`.
```

Comment:
```suggestion
`ConfigLoader` and `TemplatedConfigLoader` have been deprecated since Kedro `0.18.12` and will be removed in Kedro `0.19.0`. Refer to the [migration guide for config loaders](./config_loader_migration.md) for instructions on how to update your code to use `OmegaConfigLoader`.
```